src/main/kotlin/com/example/SaladBoard/controller/PostController.kt

```diff
@@ -28,4 +28,13 @@ class PostController(val postService: PostService) {
         postService.savePost(body = postForm)
         return PostForm(body = postForm.body)
     }
+
+    /**
+     * 投稿一覧画面を返す
+     */
+    @RequestMapping(value = ["/home"], method = [RequestMethod.GET])
```

Comment:
```suggestion
    @RequestMapping(value = ["/"], method = [RequestMethod.GET])
```
こうでもよいかなと思いました
パスを指定しなければHOMEが表示されます(http://localhost:8080)
あるいは`["/", "/home"]`とかでも